index.html

```diff
@@ -423,7 +423,8 @@ <h4>Resource limitations</h4>
 <li>Script satisfactions with a serialized <samp>scriptSig</samp> over 1650 bytes are invalid by standardness (P2SH).</li>
 <li>Script satisfactions with a witness consisting of over 100 stack elements (excluding the script itself) are invalid by standardness (P2WSH, P2SH-P2WSH).</li>
 </ul>
-Miniscript makes it easy to verify that the ability to satisfy a script is not impacted by these limits. Note that this is different from verifying whether the limits are never
+A Miniscript whose script is larger than 3600 bytes is invalid.
```

Comment:
+ @sanket1729 

To what extent do we want to restrict Miniscript to P2WSH (for now)?

The C++ implementation does make that assumption, but it doesn't seem like this website currently makes that explicit, and neither does the Rust implementation (AFAIK).

Reply:
The Rust implementation does support P2SH.
The website currently states
> For now, Miniscript is really only designed for P2WSH and P2SH-P2WSH embedded scripts

Maybe this sentence could be updated to:
> For P2WSH, a Miniscript whose script is larger than 3600 bytes is invalid.

Reply:
Ah, good.

Yes, that sounds good.

Reply:
Done